Governance/create/create-basics.md

```diff
@@ -1,57 +1,118 @@
-See [this page](../tech/filenames.md) for some tips to creating a robust filenaming convention for your datasets.
+Climate datasets can be complex and often too big to be stored in a single file.
+Raw output refers to the files generated by a model, analysis workflow, or instrument. Raw output is optimised to the tool that produced it. For example models often output all variables at a single timestep in one file. This is optimal for the model but not necessarily for analysis and long term storage. 
+Data files should be structured into a navigable directory structure that describes what different data the set contains. Ideally a directory tree should be meaningful and interpretable, with less than ~1000 individual files in a given directory. Depending on how many files are produced, implementing a directory structure early before the number of files become hard to track, is recommended. 
+Data Reference Syntax (DRS) is a naming system to be used within files, directories and metadata to identify data sets. DRS were first established by intercomparison projects such as CMIP and CORDEX based on Controlled Vocabularies (CV). While aspects of these DRS don't apply to smaller datasets they offer a useful framework to organise climate data and choosing names that are meaningful and recognisible. 
 
+File naming is also important, as confusion can easily arise if names are nit sufficinetly descriptive. A common recommendation is to name the files in a similar way to the directory structure. For example, if you have both monthly and daily data, put these into two separate sub-directories and include the frequency in the filenames. Other provenance details, such as experiment name and model, should be considered in the filename itself to reduce the risk of confusing different outputs. For example, a file called 'ocean_2014.nc' can be misunderstood very easily, but a file called 'ACCESS-ESM_historical-pacemaker_ocean_monthly_2014.nc' is much clearer and will reduce the risk of having to rerun models or misplacing irreplaceable observational data.  
+See [this page](../tech/drs-names.md) for some tips to creating a robust directory structure and filenaming convention for your datasets.
 
+{COMMENT PAOLA: I would put DMP first as backup is part of maanging the files, I would also probably make this way smaller? It feels like we're moving the focus from creating a file to managing/publishing it. I agree that all these things need to be considered from the start but a reminder should be sufficient without trying to cover them here. If we're not covering them sufficiently in other parts then we should/could move some of this text there}

```

Comment:
I think I agree, cognitively backup would be the last thing on the list.
There is a lot of value to what is here, but it is a lot and you're right, it's not all specifically about creation - but if we don't put it here and people don't start with the intention of publishing their data is there a risk they miss this if we were to move it? My inclination is to leave it as is, but we could move them all to subpages and just give a list here with links to each?

Reply:
For the moment I put backup last and leave the rest, we should maybe review this in meeting so we can all agree on what should stay here and what should be moved, might be easier that way, to other sections. I will create an issue to remind us to review this.